stdlib/public/core/Collection.swift

```diff
@@ -1143,7 +1143,8 @@ extension Collection {
   //    return isEmpty ? nil : self[startIndex]
   //  }
 
-  /// A value less than or equal to the number of elements in the collection.
+  /// Returns a value less than or equal to the number of elements in
+  /// the collection.
```

Comment:
The `A value less than...` is the correct wording of this, since `underestimatedCount` is a property.

Reply:
That's true